### PR TITLE
update comments for ModToggleDelegates from #83

### PR DESCRIPTION
**Pull request by TheMulhima**

### CODE REVIEW

**Comment by fifty-six** [comment]
Can you suppress the warnings caused by adding the Obsolete annotation? Rest looks fine.

**Comment by TheMulhima** [comment]
I'm not sure how to do that

**Comment by mandar1jn** [comment]
I have created a pr to the Mulhima's fork adding pragma directives to ignore the warnings

File: Assembly-CSharp/ModListMenu.cs [hunk 1]
```diff
@@ -276,7 +278,9 @@ private MenuScreen CreateModMenu(ModInstance modInst, ModToggleDelegates? toggle
                 ApplyChange: var applyChange
```
Comment (fifty-six) [inline Assembly-CSharp/ModListMenu.cs:278]: missing the pragma here

**Review by fifty-six — CHANGES_REQUESTED** [review]

**Comment by fifty-six** [comment]
Actually #89 does this anyways, including this pr.

**Pull request closed without merging**